src/main.rs

```diff
@@ -52,6 +52,13 @@ pub struct Prepare {
     /// It defaults to "recipe.json".
     #[clap(long, default_value = "recipe.json")]
     recipe_path: PathBuf,
+
+    /// Optional: the member in the workspace we wish to compile.
+    /// Note that this is useful in scenarios where we don't copy
+    /// the whole workspace over. Cargo will complain that it can't
+    /// find those members, so this replaces all of the members.
```

Comment:
I don't think we need to mention this here. Alternatively, we should change the language. Something along the lines of:

```
When --bin is specified, `cargo-chef` will ignore all members of the workspace that are not necessary to successfully compile the specific binary.
```